src/main/groovy/wooga/gradle/snyk/tasks/SnykInstall.groovy

```diff
@@ -14,21 +16,45 @@ import wooga.gradle.snyk.tasks.internal.SnykDownloadAsset
 
 class SnykInstall extends DefaultTask implements SnykInstallSpec {
 
-    private static final String SNYK_RELEASE_JSON_URL = "https://static.snyk.io/cli/%s/release.json"
+    static String getReleaseJsonUrl(String version) {
+        def scheme = System.getProperty("static.snyk.io.scheme", "https")
```

Comment:
Why separate this in scheme, host and port, instead of having a single property with everything together like `https://static.snyk.io:443` or something like that?

Reply:
Because it is easier to override only the specifics from the outside. I took this form the mock-server examples. One can setup multiple proxies etc. I just found it cleaner. But we can reduce this to a single value yes.

Reply:
Its cleaner for the test, but this looks cumbersome to do for the only real-world use case I can think for this, that is if they some day change the url for this static releases API. I believe its better to have some more test boilerplate in exchange of having a cleaner interface.